src/components/10-atoms/carousel/README.md

```diff
@@ -13,7 +13,7 @@ npm install @axa-ch/carousel
 ```js
```

Comment:
some general comment: in file `index.react.d.ts` of testimonials you forgot to delete row 3 (`type Variant = 'foo' | 'bar';`)